File: Java/src/Ejercicio11.java
import java.util.Scanner;

public class Ejercicio11 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Ingrese un numero: ");
        int N = sc.nextInt();
        for (int i = 0; i < N; i++) {
           System.out.println(fib(i));
        }
    }
    public static int fib(int i) {
        if(i < 2){
            return i;
        }else return fib(i-1)+fib(i-2);
    }
}

File: Kotlin/src/Ejercicio11.kt
fun main(){
    println("introduce un numero")
    val num = readln().toInt()
    for (i in 0..<num){
        println(fib(i))
    }
}

fun fib(num : Int) : Int{
    if (num <2){
        return num
    }else return fib(num-1) + fib(num-2)
}